### PR TITLE
Add calendar.import_event()

### PR DESCRIPTION
**Pull request by pschmitt**
I'm working on a CLI tool to import ICS files into GCal and noticed that the import endpoint wasn't implemented. So, here it is!

### CODE REVIEW

File: gcsa/google_calendar.py [hunk 1]
```diff
@@ -212,6 +212,27 @@ def add_quick_event(
         ).execute()
         return EventSerializer.to_object(event_json)
 
+    def import_event(self, event, **kwargs):
+        """Imports an event in the calendar
```
Comment (kuzmoyev) [inline gcsa/google_calendar.py:216]: Could you please add this line to the docs here?

    This operation is used to add a private copy of an existing event to a calendar.

So it's more clear what it does.

**Review by kuzmoyev — APPROVED** [review]
Great! Thanks for the PR.

If you can, please add the line as I mentioned in the comment. Or I'll do it myself in a free time.

File: gcsa/google_calendar.py (hunk 1 above)
Comment (pschmitt) [reply gcsa/google_calendar.py:216]: Sure. But do you mean the docstring right here, where you commented? The .rst docs? Both?

Comment (kuzmoyev) [reply gcsa/google_calendar.py:216]: I meant docstring here. But now I'm thinking it would be useful in `.rst` docs too. So both. Thanks!

Comment (pschmitt) [reply gcsa/google_calendar.py:216]: okok, chief.

**Comment by kuzmoyev** [comment]
Perfect! Thanks!

**Comment by pschmitt** [comment]
Cool. Can you please ping real quick on GH me when you release a new version with this change on pypi? Thanks!

**Comment by pschmitt** [comment]
Nevermind. I just saw #90. Will subscribe there ;)

**Comment by kuzmoyev** [comment]
v1.2.0's out :)

**Pull request merged**